.tugboat/steps/3-build.sh

```diff
@@ -6,5 +6,5 @@
 set -eux
 echo "Building..."
 
-./vendor/bin/task build
-./vendor/bin/task update
+./vendor/bin/task tugboat:build
+./vendor/bin/task tugboat:update
```

Comment:
This would probably break projects that have already customized the sync command.

@deviantintegral Do we move forward with this as a breaking change and make a release note that people should update their projects?

Reply:
@beto-aveiga added conditionals that allow for backwards compatibility.